### PR TITLE
Ngq v2

### PR DESCRIPTION
**Pull request by bhekuwenza**
@moselhy faq list updated

### CODE REVIEW

**Comment by bhekuwenza** [comment]
heey @moselhy please review this pull request

**Comment by moselhy** [comment]
image
image

The sign-in button at the top right changes when hovered over, is that intentional?

**Comment by bhekuwenza** [comment]
@moselhy pushed fix

File: client/src/components/tool/CollectData.jsx [hunk 1]
```diff
@@ -167,42 +187,30 @@ export default function CollectData() {
 										>
 											<figure className="image">
 												<img
-													src={
-														"/assets/tool/collect/thumb-kobo-monitoring-forms.png"
-													}
-													alt="kobo monitoring forms thumbnail"
+													src={"#"}
```
Comment (moselhy) [inline client/src/components/tool/CollectData.jsx:190]: check image src

File: client/public/index.html [hunk 2]
```diff
@@ -1,108 +1,97 @@
 <!DOCTYPE html>
```
Comment (moselhy) [inline client/public/index.html:1]: revert this file

File: client/src/components/FAQ.jsx [hunk 3]
```diff
@@ -64,18 +64,18 @@ export default function FAQ() {
...
 			<section>
-				<div className="section-wrap posts">
+				<div className="section-wrap posts faq">
 					<div className="intro">
 						<h1 className="section-subtitle">
 							Frequently Asked Questions
 						</h1>
 
-						<div className="FAQs-count small">
+						{/* <div className="FAQs-count small">
```
Comment (moselhy) [inline client/src/components/FAQ.jsx:73]: revert from here down

File: client/src/components/elements/NotificationLine.jsx [hunk 4]
```diff
@@ -35,13 +35,9 @@ function NotificationLine(props) {
 				enterDelay={300}
 				leaveTouchDelay={500}
 			>
-				{props?.tip?.context === "icon" ? (
-					iconElement
-				) : (
-					<span>{props?.children}</span>
-				)}
+				{props?.tip?.context === "icon" ? iconElement : props?.children}
```
Comment (moselhy) [inline client/src/components/elements/NotificationLine.jsx:38]: revert this file

File: client/src/components/layout/Header.jsx [hunk 5]
```diff
@@ -231,7 +231,7 @@ export default function Header() {
 
 	return (
 		<>
-			<Grid
+			{/* <Grid
```
Comment (moselhy) [inline client/src/components/layout/Header.jsx:234]: add back banner/popup

File: server/package-lock.json [hunk 6]
```diff
@@ -2110,27 +2110,6 @@
 				"node": "6.* || 8.* || >=10.*"
 			}
 		},
-		"node_modules/@sindresorhus/is": {
```
Comment (moselhy) [inline server/package-lock.json:2113]: revert this file

**Review by moselhy — CHANGES_REQUESTED** [review]

File: client/public/index.html (hunk 2 above)
Comment (bhekuwenza) [reply client/public/index.html:1]: @moselhy I think you should revert this, and I will pull. I did edit this file

File: client/src/components/layout/Header.jsx (hunk 5 above)
Comment (bhekuwenza) [reply client/src/components/layout/Header.jsx:234]: Launch is tomorrow. Will not need banner for this.

File: client/public/index.html (hunk 2 above)
Comment (bhekuwenza) [reply client/public/index.html:1]: Looks like an indentation issue actually.

**Pull request merged**